packages/app/src/components/SearchResultsCard/index.js

```diff
@@ -24,19 +21,18 @@ const SearchResultsCard = () => {
               <td><h1><strong>Patient Name</strong></h1></td>
               <td><h1><strong>Received Date</strong></h1></td>
             </tr>
-            { SearchResultData && SearchResultData.map((result) => (
-              <tr key={result.id} >
-                <td className="nom-name">
-                <Link target={"_blank"} to={`/nomination/${result.id}`}>
-                    {result.nominationName}
-                </Link>
-                </td>
-                <td>{result.providerName}</td>
-                <td>{result.patientName}</td>
-                <td>{result.dateReceived}</td>
-              </tr>
-              )
-            )}
+            {SearchResultData
```

Comment:
This can be done using optional chaining instead of a ternary: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining Try it out and let me know if you need help.

Reply:
Looks like you fixed this in the healthProviderSearch file but not this one.